### PR TITLE
Fix CI intro

### PR DESCRIPTION
**Pull request by simleo**
Fixes the "job names" section in `contributing/ci-introduction.txt`

### CODE REVIEW

File: contributing/ci-introduction.txt [hunk 1]
```diff
@@ -62,21 +62,21 @@ Multiple labels are used in the PR reviewing process:
...
-    `origin/dev_5_0`,
-  * the ``merge`` jobs build from the tip of the development branch with all
-    PRs merged using :ref:`scc merge` with the `org` default filter set,
-  * the ``breaking`` jobs build from the tip of the development branch with
-    all PRs labeled as ``breaking`` merged using :ref:`scc merge`,
-  * the ``release`` jobs build from and optionally create a tag at the tip of
+All core OME job names take the form
+``COMPONENT-VERSION-TYPE-DESCRIPTION``, where:
+
+- ``COMPONENT`` is the core OME component, e.g. OMERO or BIOFORMATS.
```
Comment (hflynn) [inline contributing/ci-introduction.txt]: BIOFORMATS -> Bio-Formats please. Otherwise this is good to merge, looks fine on staging thanks.

**Comment by hflynn** [comment]
Will release contributing does once this is merged so the latest version is live.

File: contributing/ci-introduction.txt (hunk 1 above)
Comment (simleo) [inline contributing/ci-introduction.txt]: It refers to (part of) the CI tag, as in `BIOFORMATS-5.1-latest`. Maybe it's better if I quote it?

**Comment by simleo** [comment]
I've reworded the COMPONENT item to distinguish between tags and project names. Also, formatting should now be more consistent, avoiding confusion between placeholders and values.

**Comment by hflynn** [comment]
All good on staging thanks, merging and releasing.

**Pull request merged**